experimental/token/token.go

```diff
@@ -322,11 +322,10 @@ func (t Token) Children() *Cursor {
 	}
 
 	if impl := t.nat(); impl != nil {
-		start, end := t.StartEnd()
+		start, _ := t.StartEnd()
 		return &Cursor{
 			withContext: t.withContext,
-			start:       start.id + 1, // Skip the start!
-			end:         end.id,
+			idx:         int(start.id), // Skip the start!
```

Comment:
This comment is a big confusing now. The + 1 is now implied, but it's... not at all obvious what's going on here.

I wonder if we should have something like `ID.index()` which does the `- 1`...? Would be good for a followup, probably.